tests/ui/non_std_lazy_static/non_std_lazy_static_fixable.rs

```diff
@@ -0,0 +1,72 @@
+//@aux-build:once_cell.rs
+//@aux-build:lazy_static.rs
+
+#![warn(clippy::non_std_lazy_statics)]
```

Comment:
This is missing a `lazy_static!` test case thats both local and within the required MSRV.